src/app/candidate-management/candidate-management-detail/candidate-management-detail.component.ts

```diff
@@ -231,6 +252,7 @@ export class CandidateManagementDetailComponent implements OnInit, AfterViewInit
       }
 
       this.disabled = true;
+      this.router.navigate(['./candidate/detail', this.candidate.name]);
```

Comment:
The route change will also trigger a reload of the candidate so the lines 242 and 243 become somewhat redundant.
Also use the routing constants defined in the constants module consistently and not just for the routing path definitions.